<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autofloder Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="combo_studios">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Studio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reading</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_code">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Campaign ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_campaign_id">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_counter">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Counter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_approve_all">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>32</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Approve All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_all">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>32</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_refresh">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>32</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_single">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Single</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_multiple">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_image">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>550</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Image</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scroll_area">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>507</width>
       <height>447</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_load_refences">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load References</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeRefences">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>701</width>
      <height>451</height>
     </rect>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">6</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Incio</string>
    </property>
    <addaction name="action_open_image"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
